scripts/install.sh

```diff
@@ -63,54 +65,50 @@ shift $(expr $OPTIND - 1) # remove options from positional parameters
 # Overwrite default properties
 CONFIG_DIR_PATH=${config_dir_path:-$CONFIG_DIR_PATH_DEFAULT}
 NETWORK=${network:-$NETWORK_DEFAULT}
-CONTRACTS_ONLY=${contracts_only:-false}
+REUSE_CONTRACTS=${reuse_contracts:-false}
+
+echo "reuse contracts::" ${REUSE_CONTRACTS}
 
 # Run script
 printf "${LOG_START}Starting installation...${LOG_END}"
 
 printf "Config dir path: $CONFIG_DIR_PATH\n"
 printf "Network: $NETWORK"
 
-cd $KEEP_CORE_SOL_PATH
+cd $KEEP_BEACON_SOL_PATH
 
-printf "${LOG_START}Installing NPM dependencies...${LOG_END}"
-npm install
+printf "${LOG_START}Installing YARN dependencies...${LOG_END}"
+# rm -rf node_modules/
+yarn install
 
-if [ "$NETWORK" == "local" ]; then
+if [ "$NETWORK" == "development" ]; then
     printf "${LOG_START}Unlocking ethereum accounts...${LOG_END}"
     KEEP_ETHEREUM_PASSWORD=$KEEP_ETHEREUM_PASSWORD \
-        npx truffle exec scripts/unlock-eth-accounts.js --network $NETWORK
+        npx hardhat run scripts/unlock-eth-accounts.ts --network $NETWORK
 fi
 
-printf "${LOG_START}Migrating contracts...${LOG_END}"
-rm -rf build/
+printf "${LOG_START}Building contracts...${LOG_END}"
+# rm -rf build && rm -rf cache && rm -rf typechain
```

Comment:
Uncomment?

Reply:
Part of: [2dcb842](https://github.com/keep-network/keep-core/pull/3026/commits/2dcb842ce661077ccb2491e425aa08f4112bbdbe)